### ISSUE TITLE
Add robustness CIFAR test sets 

### ISSUE THREAD

**Issue by dustinvtran** [google/uncertainty-baselines#60]
> @dustinvtran 
> For cifar 10.1, add nll / accuracy / ce. For cifar c, add mean nll, mce, and mean ce. similarly cifar p.
>
> It looks like adding extra validation sets with the `model.fit 
API is fairly involved (https://stackoverflow.com/questions/47731935/using-multiple-validation-sets-with-keras). We're planning to move to custom training loops anyways, so I'll get started on that first.
>
> This issue should be moved to Robustness Metrics' issue tracker once open-sourced.

https://github.com/google/edward2/issues/96
